tests/action_tests.py

```diff
@@ -31,8 +31,9 @@ def test_provision_with_missing_data():
     boompy.set_auth("account_id", "username", "password")
     boompy.actions.provisionPartnerCustomerAccount(fake_data)
 
+@mock.patch.object(requests.Session, "get")
```

Comment:
I am a fan of habitually adding `autospec=True` to all `mock.patch` and `mock.patch.object` calls, except when you can't (doesn't work on classmethods and staticmethods.)